docs/general/builtins/bit-operators.rst

```diff
@@ -0,0 +1,40 @@
+.. highlight:: psql
+
+.. _bit-operators:
+
+=============
+Bit operators
+=============
+
+Bit :ref:`operators <gloss-operator>` perform bitwise operations on numeric
+integral values and :ref:`bit <data-type-bit>` strings:
+
+========  ========================
+Operator  Description
+========  ========================
+``&``     Bitwise AND of operands.
+``|``     Bitwise OR of operands.
+``#``     Bitwise XOR of operands.
+========  ========================
+
+Here's an example that uses all of the available bit operators::
+
+    cr> select 1 & 2 | 3 # 4 AS n;
```

Comment:
Would it be too much to have an example with bits (`select B'100' # B'011'`)?

Reply:
Thx, actually sport an issue - new doc file was not added to tests
See https://github.com/crate/crate/pull/13288